tests/rptest/services/redpanda.py

```diff
@@ -169,8 +167,13 @@ def start(self, nodes=None, clean_nodes=True):
         self._client = KafkaAdminClient(bootstrap_servers=self.brokers_list(),
                                         **security_settings)
 
-    def _create_initial_topics(self):
-        client = self._client_type(self)
+        self._create_initial_topics(security_settings)
+
+    def _create_initial_topics(self, security_settings):
+        user = security_settings.get("sasl_plain_username")
+        passwd = security_settings.get("sasl_plain_password")
+
+        client = self._client_type(self, user=user, passwd=passwd)
         for spec in self._topics:
```

Comment:
looks good. for future reference this would have been a good change to break out into a separate commit.